### PR TITLE
`Development`: Remove remaining mentions of JIRA, Bitbucket and Bamboo in the documentation

### PR DESCRIPTION
**Pull request by MaximilianAnzinger**
#### General

- [x] I chose a title conforming to the [naming conventions for pull requests](https://docs.artemis.cit.tum.de/dev/development-process/#naming-conventions-for-github-pull-requests).

#### Server
- [x] I **strictly** followed the [server coding and design guidelines](https://docs.artemis.cit.tum.de/dev/guidelines/server/).
- [x] I documented the Java code using JavaDoc style.

#### Client
- [x] I **strictly** followed the [client coding and design guidelines](https://docs.artemis.cit.tum.de/dev/guidelines/client/).
- [x] I documented the TypeScript code using JSDoc style.

### Description

Removes remaining artifacts of Atlassian.

#### Code Review
- [x] Code Review 1
- [ ] Code Review 2

### CODE REVIEW

**Review by rabeatwork — APPROVED** [review]
maintainer approved

File: src/main/java/de/tum/in/www1/artemis/service/exam/StudentExamService.java [hunk 1]
```diff
@@ -809,7 +840,7 @@ private StudentExam generateIndividualStudentExam(Exam exam, User student) {
     @Transactional // TODO: NOT OK --> remove @Transactional
     public List<StudentExam> generateStudentExams(final Exam exam) {
         final var existingStudentExams = studentExamRepository.findByExamId(exam.getId());
-        // https://jira.spring.io/browse/DATAJPA-1367 deleteInBatch does not work, because it does not cascade the deletion of existing exam sessions, therefore use deleteAll
+        // deleteInBatch does not work, because it does not cascade the deletion of existing exam sessions, therefore use deleteAll
         studentExamRepository.deleteAll(existingStudentExams);
```
Comment (Strohgelaender) [inline src/main/java/de/tum/in/www1/artemis/service/exam/StudentExamService.java:844]: This has nothing to do with our Jira instance, but instead links to a Spring issue.

**Review by Strohgelaender — COMMENTED** [review]
lgtm otherwise

File: src/main/java/de/tum/in/www1/artemis/service/exam/StudentExamService.java (hunk 1 above)
Comment (krusche) [reply src/main/java/de/tum/in/www1/artemis/service/exam/StudentExamService.java:844]: I think it's ok and we don't need the link to the JIRA Ticket

**Pull request merged**